source/app/api.py

```diff
@@ -154,6 +154,11 @@ def add_disk(disk):
                           'asd_id': asd_id,
                           'log_level': 'info',
                           'port': port}
+            
+            if config.data['main'].has_key('__sync_dont_use'):
+                asd_config['__sync_dont_use'] = config.data['main']['__sync_dont_use']
```

Comment:
It might be an idea to make it more generic. E.g. add an "extra_parameters" key in the config file (next to main and network) that is used by the code, adding every key/value pair in that "extra_parameters" key to the asd_config.